libr/crypto/p/crypto_serpent.c

```diff
@@ -50,11 +50,14 @@ static bool update(RCrypto *cry, const ut8 *buf, int len) {
 	}
 
 	// Construct ut32 blocks from byte stream
-	for (j = 0; j < size/4; j++) {
-		ibuf[j] = r_read_le32(&buf[4*j]);
+	for (j = 0; j < len/4; j++) {
+		ibuf[j] = r_read_le32 (&buf[4*j]);
+	}
+	if (len & 0x3) {
+		ut8 tail[4] = {0}; // Zero padding
+		memcpy (tail, buf + (len & ~0x3), len & 0x3);
+		ibuf[len/4] = r_read_le32 (tail);
```

Comment:
add spaces around /, * in all the file.

also &buf[4*j] is the same as buf + (4*j) , which is imho cleaner and easy to read